ci-operator/step-registry/hypershift/openstack/e2e/execute/hypershift-openstack-e2e-execute-commands.sh

```diff
@@ -43,14 +30,13 @@ hack/ci-test-e2e.sh \
         --e2e.previous-release-image=${OCP_IMAGE_PREVIOUS} \
         --e2e.pull-secret-file=/etc/ci-pull-credentials/.dockerconfigjson \
         --e2e.node-pool-replicas=2 \
-        --e2e.base-domain="${BASE_DOMAIN}" \
+	--e2e.aws-credentials-file=${CLUSTER_PROFILE_DIR}/.awscred \
+	--e2e.base-domain=origin-ci-int-aws.dev.rhcloud.com \
         --test.run='^TestCreateCluster$' \
 	--test.parallel=1 \
         --e2e.platform="OpenStack" \
 	--e2e.ssh-key-file="${CLUSTER_PROFILE_DIR}/ssh-publickey" \
         --e2e.openstack-credentials-file="${SHARED_DIR}/clouds.yaml" \
-	--e2e.openstack-ingress-floating-ip="${INGRESS_FIP}" \
```

Comment:
we might need to add this back eventually

Reply:
no because we'll run tests in parallel, and we'll use the router-default IP that is created by CCM.